extensions/wikia/ImageReview/modules/PromoteImage/PromoteImageReviewHelper.class.php

```diff
@@ -26,17 +26,17 @@ public function updateImageState($images, $action = '') {
 		$taskAdditionList = array();
 
 		$sqlWhere = array(
-			ImageReviewStatuses::STATE_APPROVED_AND_TRANSFERRING => array(),
-			ImageReviewStatuses::STATE_REJECTED => array(),
-			ImageReviewStatuses::STATE_QUESTIONABLE => array(),
+			ImageReviewStates::APPROVED_AND_TRANSFERRING => array(),
+			ImageReviewStates::REJECTED => array(),
+			ImageReviewStates::QUESTIONABLE => array(),
 		);
 
 		foreach( $images as $image ) {
-			if ($image['state'] == ImageReviewStatuses::STATE_APPROVED) {
+			if ($image['state'] == ImageReviewStates::APPROVED) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "state" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)